include/functions/utility.hpp

```diff
@@ -110,4 +110,30 @@ bool is_power_of_10(const std::string& num){
     return true;    // first digit is 1 and the following digits are all 0
 }
 
+
+/*
+    conver_to_base_64
+    ----------------------
+    Converts a string represented BigInt into the Base64 magnitudes
+*/
+
+std::vector<unsigned long long> convert_to_base_64(std::string& num) {
+    std::vector<unsigned long long> magnitude;
+    auto start_it = num.size() - 1; // Starting from Backward direction
+    auto stop_it = num.size();
+    uint64_t component = 0;
+    while(start_it != -1) {
+        try {
+            std::string num_slice = num.substr(start_it, stop_it);
+            component = std::stoull(num_slice);
+        } catch (std::out_of_range& e) { //Using out_of_range to determine overflow
```

Comment:
This logic wouldn't work. What we want is to divide the number by the largest power of 2<sup>64</sup> that gives a non-zero quotient, push that in an array, use the remainder as the new number and repeat the process with decreasing powers until we reach 1 (0th power). Then return the reversed array.

Reply:
I was wondering how can we divide a number even with 2<sup>64</sup>, the dividend will anyway overflow if we are going to do these in the traditional way. So i am thinking it must all be done in strings rather than converting the numbers into integer chunks or something related.

Reply:
Yes, we'll need to do string-based division first to convert to base 2<sup>64</sup>. I'm working on removing as much code as possible while still having string division and remainder work. But it'll take some time as I'm busy with exams currently. In the meanwhile, if you have another suggestion, or would like to work on this then let me know.

Reply:
Yes, i will modify it to use string divisions instead.